test/wasm2c/spec/threads/atomic.txt

```diff
@@ -0,0 +1,6 @@
+;;; TOOL: run-spec-wasm2c
+;;; STDIN_FILE: test/wasm2c/spec/threads/atomic_supported.wast
```

Comment:
In practice this can be done directly in the .txt file (instead of needing to reference a separate file); e.g. https://github.com/WebAssembly/wabt/blob/1.0.30/test/wasm2c/old-spec/select.txt